linera-base/src/crypto/signer.rs

```diff
@@ -80,6 +80,13 @@ mod in_mem {
     #[derive(Clone)]
     pub struct InMemorySigner(Arc<RwLock<InMemSignerInner>>);
 
+    #[cfg(not(with_getrandom))]
+    impl Default for InMemorySigner {
```

Comment:
Why exactly the default trait should be implemented only in that case for the `InMemorySigner`?
The restriction `with_getrandom` seems arbitrary.

Reply:
This is solving a clippy warning. In the other case, `new` has an extra argument!